A3A/addons/core/functions/Missions/fn_LOG_Airdrop.sqf

```diff
@@ -2,7 +2,7 @@
 FIX_LINE_NUMBERS()
 
 //Mission: Catch airdrop
-if (!isServer and hasInterface) exitWith{};
+if (isServer and hasInterface) exitWith{};
```

Comment:
You are doing this intentionally? Just want to make sure...

Reply:
Forgot to remove if (isServer and hasInterface) exitWith{}; from missions after testing, needs to be changed back to if (!isServer and hasInterface) exitWith{}; after everything is ready